src/commands/clone-backend.ts

```diff
@@ -0,0 +1,62 @@
+import {BaseCommand} from '../lib'
+import {flags} from '@oclif/command'
+import {cli} from 'cli-ux'
+
+const RESTORE_MUTATION  = `
+mutation($uid: String!) {
+  restore(uid: $uid) {
+    response { code message restoreId }
+  }
+}`
+
+export default class CloneBackend extends BaseCommand {
+    static description = 'Clone into a backend by source backend ID'
+
+    static hidden = true;
+
+    static examples = [
+      '$ slash-graphql clone-backend -e https://clone.cloud.dgraph.io/graphql -t <apiToken> --source <source backend id or url>',
```

Comment:
let's do id in the example?

Reply:
Users don't have access to deployment ID yet.